src/components/CourtsTable/SwitchButton.tsx

```diff
@@ -0,0 +1,59 @@
+import { Flex, FormControl, FormLabel, Switch } from "@chakra-ui/react";
+import React from "react";
+import { useUpdateCourtMutation } from "../../redux/api/courtsApi";
+import { ICourt } from "@/interfaces/courtData";
+interface Props {
+  court: ICourt;
+}
+
+const SwitchButton = ({ court }: Props) => {
+  const [updateCourt] = useUpdateCourtMutation();
+
+  const handSwitchBtnState = () => {
+    updateCourt({ ...court, ...{ updatedAt: new Date().toString(), isHidden: !court.isHidden } });
```

Comment:
I think it is better to do the date update in the backend

Reply:
Yes, you are right! The functionality of updating the data has been done at the backend, it is duplicated here. Thanks guys!

Reply:
Yes, you are right! The functionality of updating the date has been done at the backend, it is duplicated here. Thanks guys!